docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -357,9 +357,7 @@ $ journalctl -fu dm-event.service
 If you run into repeated problems with thin pool, you can use the
 `dm.min_free_space` option to tune the Engine behavior. This value ensures that
 operations fail with a warning when the free space is at or near the minimum.
-For information, see <a
-href="../../../reference/commandline/dockerd/#storage-driver-options"
-target="_blank">the storage driver options in the Engine daemon reference</a>.
+For information, see [the storage driver options in the Engine daemon reference](../../reference/commandline/dockerd.md#storage-driver-options).
```

Comment:
This link was a HTML link, so that the "target=blank" could be set; can you try if it was actually broken? (Sorry, not near a computer)